xml/System.Security.Authentication/SslProtocols.xml

```diff
@@ -39,6 +39,10 @@
     <summary>Defines the possible versions of <see cref="T:System.Security.Authentication.SslProtocols" />.</summary>
     <remarks>
       <format type="text/markdown"><![CDATA[  
+      
+## Remarks
+
+  During the handshake, the client sends a list of SSL versions and the server chooses from that list the best matching version. 
```

Comment:
NIT: 
the server chooses from that list the best matching version.  --> the server chooses the best matching version from that list.